pkg/services/alerting/notifiers/pushover.go

```diff
@@ -207,34 +208,36 @@ func NewPushoverNotifier(model *models.AlertNotification) (alerting.Notifier, er
 		return nil, alerting.ValidationError{Reason: "API token not given"}
 	}
 	return &PushoverNotifier{
-		NotifierBase:  NewNotifierBase(model),
-		UserKey:       userKey,
-		APIToken:      APIToken,
-		Priority:      priority,
-		Retry:         retry,
-		Expire:        expire,
-		Device:        device,
-		AlertingSound: alertingSound,
-		OkSound:       okSound,
-		Upload:        uploadImage,
-		log:           log.New("alerting.notifier.pushover"),
+		NotifierBase:     NewNotifierBase(model),
+		UserKey:          userKey,
+		APIToken:         APIToken,
+		AlertingPriority: alertingPriority,
+		OkPriority:       okPriority,
```

Comment:
Standard Go style dictates that initialisms be written in consistent case:
```suggestion
		OKPriority:       okPriority,
```